src/pilot/ironic.patch

```diff
@@ -1,6 +1,99 @@
+ # service. (list value)
+ #enabled_management_interfaces = ipmitool
+-enabled_management_interfaces=fake,idrac,ilo,ipmitool,noop,redfish
++enabled_management_interfaces=fake,idrac-redfish,idrac-wsman,ilo,ipmitool,noop,redfish
+ 
+ # Default management interface to be used for nodes that do
+ # not have management_interface field set. A complete list of
+@@ -182,6 +182,7 @@
+ # enumerating the "ironic.hardware.interfaces.management"
+ # entrypoint. (string value)
+ #default_management_interface = <None>
++default_management_interface=idrac-wsman
+ 
+ # Specify the list of network interfaces to load during
+ # service initialization. Missing network interfaces, or
+@@ -223,7 +224,7 @@
+ # type will have the same set of enabled power interfaces on
+ # every ironic-conductor service. (list value)
+ #enabled_power_interfaces = ipmitool
+-enabled_power_interfaces=fake,idrac,ilo,ipmitool,redfish
++enabled_power_interfaces=fake,idrac-redfish,idrac-wsman,ilo,ipmitool,redfish
+ 
+ # Default power interface to be used for nodes that do not
+ # have power_interface field set. A complete list of power
+@@ -231,6 +232,7 @@
+ # enumerating the "ironic.hardware.interfaces.power"
+ # entrypoint. (string value)
+ #default_power_interface = <None>
++default_power_interface=idrac-wsman
+ 
+ # Specify the list of raid interfaces to load during service
+ # initialization. Missing raid interfaces, or raid interfaces
+@@ -247,13 +249,14 @@
+ # enabled raid interfaces on every ironic-conductor service.
+ # (list value)
+ #enabled_raid_interfaces = agent,no-raid
+-enabled_raid_interfaces=idrac,no-raid
++enabled_raid_interfaces=idrac-wsman,no-raid
+ 
+ # Default raid interface to be used for nodes that do not have
+ # raid_interface field set. A complete list of raid interfaces
+ # present on your system may be found by enumerating the
+ # "ironic.hardware.interfaces.raid" entrypoint. (string value)
+ #default_raid_interface = <None>
++default_raid_interface=idrac-wsman
+ 
+ # Specify the list of rescue interfaces to load during service
+ # initialization. Missing rescue interfaces, or rescue
+@@ -319,7 +322,7 @@
+ # type will have the same set of enabled vendor interfaces on
+ # every ironic-conductor service. (list value)
+ #enabled_vendor_interfaces = ipmitool,no-vendor
+-enabled_vendor_interfaces=idrac,ipmitool,no-vendor
++enabled_vendor_interfaces=idrac-wsman,ipmitool,no-vendor
+ 
+ # Default vendor interface to be used for nodes that do not
+ # have vendor_interface field set. A complete list of vendor
+@@ -327,6 +330,7 @@
+ # enumerating the "ironic.hardware.interfaces.vendor"
+ # entrypoint. (string value)
+ #default_vendor_interface = <None>
++default_vendor_interface=idrac-wsman
+ 
+ # DEPRECATED: Used if there is a formatting error when
+ # generating an exception message (a programming error). If
+@@ -701,6 +705,7 @@
+ 
+ # Seconds to wait for a response from a call. (integer value)
+ #rpc_response_timeout = 60
++rpc_response_timeout = 600
```

Comment:
Is the change in the line above intended?

Reply:
Yes, it addresses a [known issue](https://docs.openstack.org/ironic/latest/admin/drivers/idrac.html#vendor-passthru-timeout). @himanshusingh2 observed `assigne_role.py` encounter it during BIOS settings configuration.

Reply:
Thanks!